src/linear_algebra/clifford_algebra.lean

```diff
@@ -0,0 +1,153 @@
+/-
+Copyright (c) 2020 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser, Utensil Song.
+-/
```

Comment:
I don't know whether to inherit the copyright from `exterior_algebra` here

Reply:
I'm okay with it as is.